core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -92,12 +92,13 @@
                  translate="I18N_TOPNAV_GET_STARTED">
               </a>
             </li>
+            <!-- TODO(#12070): Commented out until new pages/modals are made.
             <li class="nav-item">
               <a class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-playbook-link "
                  href="/teach#participation"
                  translate="I18N_TOPNAV_PARTICIPATION_PLAYBOOK">
               </a>
-            </li>
+            </li> -->
```

Comment:
Please move the code in the issue and remove it from here. (Avoid having unwanted/unused comments/codes in the codebase.)

Reply:
done.